packages/blocks/common/src/lib/helpers/index.ts

```diff
@@ -82,11 +82,11 @@ export function createCustomApiCallAction({
         displayName: 'Headers',
         description:
           'Authorization headers are injected automatically from your connection.',
-        required: true,
+        required: false,
```

Comment:
Fixes https://linear.app/openops/issue/OPS-1900/allow-empty-headers-and-query-params-in-custom-api-actions